Source/BSN.Commons.PresentationInfrastructure/Responses/GenericResponseBase.cs

```diff
@@ -1,13 +0,0 @@
-﻿namespace BSN.Commons.Responses
```

Comment:
For backward compatibility issues, please do not remove these classes

Reply:
It's a breaking change. we cant do this.

Reply:
Where is contructor?